Dockerfile

```diff
@@ -1,8 +1,17 @@
-FROM golang:1.11.5-alpine
+FROM golang:1.11.5-alpine as base
 
 ENV GO111MODULE on
 
 RUN apk --no-cache add gcc g++ make ca-certificates git
-
 RUN mkdir /weaver
+ADD . /weaver
 WORKDIR /weaver
+
+RUN make setup
+RUN make build
+
+FROM golang:1.11.2-alpine
```

Comment:
cant this be base alpine instead of golang. since the build is complete

Reply:
The changes are done here @irfn https://github.com/gojektech/weaver/blob/master/Dockerfile

Reply:
Yep, it can, thanks @irfn